tools/clang/test/HLSLFileCheck/hlsl/workgraph/entry_node_attr_on_non_compute_stage.hlsl

```diff
@@ -0,0 +1,21 @@
+// RUN: %dxc -T ps_6_6 -E main %s | FileCheck %s
+
+struct loadStressRecord {
+    uint x : SV_DispatchGrid;
+};
+
+void loadStressWorker(NodeOutput<loadStressRecord> x) {
+    return;
+}
+
+[Shader("node")]
+[NodeMaxDispatchGrid(3, 1, 1)]
+[NumThreads(16, 1, 1)]
+void main(
+    // CHECK: error: entry type 'pixel' from profile 'ps_6_6' conflicts with shader attribute type 'node' on entry function 'main'.
```

Comment:
This needs to be a filecheck test because filecheck tests provide a way to input the target shader stage. I need the shader stage to be valid, and to not be compute, in order to run this test. AFAIK, there's no way to specify a valid, non-compute shader stage in a verifier test.

Reply:
You can use %dxc  -verify in lit test which has all the options.